lib/WorkflowMain.groovy

```diff
@@ -71,12 +71,6 @@ class WorkflowMain {
 
         // Check AWS batch settings
         NfcoreTemplate.awsBatch(workflow, params)
-
```

Comment:
Why do you want to remove this? Do you think the error message is not useful?

Reply:
when you want to specify a `step` in a project and want to make use of the automatic detection of the intermediate csv. If you don't provide an input then it fails.

Reply:
I think this is template code, we should try to fix it then. Can you send me an example command?